scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -103,7 +103,7 @@ def get_memballoon_type():
     </memballoon>"""
 
     return """    <memballoon model='virtio' autodeflate='on'>
-      <address type='pci' slot='0x05'/>
+      <address type='pci' slot='0x15'/>
```

Comment:
@nicolasbock with "more intelligent" I meant some method that returns the next free slot for that host and this method should be used for each device that is added to the VM configuration. Just moving the IDs by 0x20 only moves the issue to later or to bigger installation. I recommend to find a solution that prevents the collisions at all.